test/pos.test.ts

```diff
@@ -36,16 +36,6 @@ describe('matched-positions', async () => {
             22,
             "[border=\\"2\\"]",
           ],
-          [
-            23,
-            37,
-            "[border=\\"[&_span]:white\\"]",
-          ],
-          [
-            46,
-            65,
-            "[hover=\\"[&>span]:text-white\\"]",
-          ],
```

Comment:
Please include presetArbitrary on this test too. It tests highlighting arbitrary variants in attributify context.